CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+## 0.2.0 / 2016-03-19
+
+NOTE: This release renames `statsd_bridge` to `statsd_exporter`
+
+* [CHANGE] Migrate logging to prometheus/log. (#14)
+* [CHANGE] Document use-case without any StatsD server. (#16)
+* [CHANGE] New Dockerfile using alpine-golang-make-onbuild base image (#17)
+* [IMPROVEMENT] Allow configuration of UDP read buffer (#22)
+* [CHANGE] Move from client_golang/model to common/model (#23)
+* [BUGFIX] allow metrics with dashes when mapping (#24)
+* [IMPROVEMENT] add root endpoint with redirect (#25)
+* [CHANGE] rename bridge to exporter (#26)
+* [BIGFIX] fix documentation for docker udp port publishing (#31)
```

Comment:
`BUGFIX` – but this doesn't affect the program itself, so generally don't include this in changelogs.